index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
+    <div class="header-cont">
+      <div class="nav-cont">
+        <div class="header-img"><img src="images/image_geometry_1.png" alt="">
+        </div>
+        <nav class="nav-bar">
+          <i class="fa-solid fa-bars fa-xl"></i>
+      </div>
+      <header>
+        <h1 class="header-h1">
+          Fuad <br> Nabiyev
+        </h1>
```

Comment:
- [ ] it would be better if you add font-family and right font-size as in the Figma design in order to. match Figma design 
<img width="158" alt="Screen Shot 2022-09-21 at 17 43 47" src="https://user-images.githubusercontent.com/61790587/191563138-7f526d9b-b07d-4a63-a4c0-b16034b12496.png">